src/Form/GoldenBookType.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\GoldenBook;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+
+class GoldenBookType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('name', TextType::class, ['label' => 'Prénom'])
+            ->add('departureDate', DateType::class, ['label' => 'Date de départ'])
+            ->add('commentary', TextType::class, ['label' => 'commentaire'])
+            ->add('villa', null, ['choice_label'=>'name'])
```

Comment:
You should Type your Form fields and...
This one should probably be an EntityType.